pkg/generators/forwarding/fluentd/output_fluentd_buffer.go

```diff
@@ -0,0 +1,122 @@
+package fluentd
+
+import (
+	"fmt"
+
+	loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+)
+
+const (
+	// Buffer size defaults
+	defaultOverflowAction = "block"
+
+	// Flush buffer defaults
+	defaultFlushThreadCount = "2"
+	defaultFlushMode        = "interval"
+	defaultFlushInterval    = "1s"
+	defaultFlushAtShutdown  = false
+
+	// Retry buffer to output defaults
+	defaultRetryWait        = "1s"
+	defaultRetryType        = "exponential_backoff"
+	defaultRetryMaxInterval = "300s"
+
+	// Output fluentdForward default
+	fluentdForwardChunkLimitSize = "1m"
+	fluentdForwardOverflowAction = "throw_exception"
```

Comment:
I believe if you look at the resolution of the env vars  that this would also be "block" and and not throw

Reply:
Good catch, looking back I was misguided by looking on `templates.go` for the default but should have looked on `run.sh`.